content/content.json

```diff
@@ -125,6 +125,13 @@
       "title": "How to Set Up Goose on Windows",
       "link": "https://dev.to/lymah/how-to-set-up-goose-on-windows-29dd",
       "contributor": "Lymah123"
+    },
+    {
+      "type": "blog",
+      "thumbnail": "./img/Using .goosehints Files with Goose.png",
```

Comment:
What a great blog post!!! Thank you for writing this. One thing we may need to change is the image name with spaces and symbols that will likely cause your image to not render properly. Could you try naming your image "using-goosehints-blog-post" and then referring to it as 

`"thumbnail": "./img/using-goosehints-blog-post",`

Thank you so much! Can't wait to get this merged in!

Reply:
I have addressed the requested changes

Thank you so much